packages/core/src/common/_colors.scss

```diff
@@ -111,67 +111,67 @@ interface design — take a look at [core colors](#colors.core) instead.
 Styleguide colors.extended
 */
 
-$vermilion1: #9e2b0e;
-$vermilion2: #b83211;
-$vermilion3: #d13913;
-$vermilion4: #eb532d;
-$vermilion5: #ff6e4a;
-
-$rose1: #a82255;
-$rose2: #c22762;
-$rose3: #db2c6f;
-$rose4: #f5498b;
-$rose5: #ff66a1;
-
-$violet1: #5c255c;
-$violet2: #752f75;
-$violet3: #8f398f;
-$violet4: #a854a8;
-$violet5: #c274c2;
-
-$indigo1: #5642a6;
-$indigo2: #634dbf;
-$indigo3: #7157d9;
-$indigo4: #9179f2;
-$indigo5: #ad99ff;
-
-$cobalt1: #1f4b99;
-$cobalt2: #2458b3;
-$cobalt3: #2965cc;
-$cobalt4: #4580e6;
-$cobalt5: #669eff;
-
-$turquoise1: #008075;
-$turquoise2: #00998c;
-$turquoise3: #00b3a4;
-$turquoise4: #14ccbd;
-$turquoise5: #2ee6d6;
-
-$forest1: #1d7324;
-$forest2: #238c2c;
-$forest3: #29a634;
-$forest4: #43bf4d;
-$forest5: #62d96b;
-
-$lime1: #728c23;
-$lime2: #87a629;
-$lime3: #9bbf30;
-$lime4: #b6d94c;
-$lime5: #d1f26d;
-
-$gold1: #a67908;
-$gold2: #bf8c0a;
-$gold3: #d99e0b;
-$gold4: #f2b824;
-$gold5: #ffc940;
-
-$sepia1: #63411e;
-$sepia2: #7d5125;
-$sepia3: #96622d;
-$sepia4: #b07b46;
-$sepia5: #c99765;
-
-@import "color-aliases";
+$vermilion1: #9e2b0e !default;
+$vermilion2: #b83211 !default;
+$vermilion3: #d13913 !default;
+$vermilion4: #eb532d !default;
+$vermilion5: #ff6e4a !default;
+
+$rose1: #a82255 !default;
+$rose2: #c22762 !default;
+$rose3: #db2c6f !default;
+$rose4: #f5498b !default;
+$rose5: #ff66a1 !default;
+
+$violet1: #5c255c !default;
+$violet2: #752f75 !default;
+$violet3: #8f398f !default;
+$violet4: #a854a8 !default;
+$violet5: #c274c2 !default;
+
+$indigo1: #5642a6 !default;
+$indigo2: #634dbf !default;
+$indigo3: #7157d9 !default;
+$indigo4: #9179f2 !default;
+$indigo5: #ad99ff !default;
+
+$cobalt1: #1f4b99 !default;
+$cobalt2: #2458b3 !default;
+$cobalt3: #2965cc !default;
+$cobalt4: #4580e6 !default;
+$cobalt5: #669eff !default;
+
+$turquoise1: #008075 !default;
+$turquoise2: #00998c !default;
+$turquoise3: #00b3a4 !default;
+$turquoise4: #14ccbd !default;
+$turquoise5: #2ee6d6 !default;
+
+$forest1: #1d7324 !default;
+$forest2: #238c2c !default;
+$forest3: #29a634 !default;
+$forest4: #43bf4d !default;
+$forest5: #62d96b !default;
+
+$lime1: #728c23 !default;
+$lime2: #87a629 !default;
+$lime3: #9bbf30 !default;
+$lime4: #b6d94c !default;
+$lime5: #d1f26d !default;
+
+$gold1: #a67908 !default;
+$gold2: #bf8c0a !default;
+$gold3: #d99e0b !default;
+$gold4: #f2b824 !default;
+$gold5: #ffc940 !default;
+
+$sepia1: #63411e !default;
+$sepia2: #7d5125 !default;
+$sepia3: #96622d !default;
+$sepia4: #b07b46 !default;
+$sepia5: #c99765 !default;
```

Comment:
hmm i wonder if `!default` is correct here. these colors are carefully calibrated and we shouldn't encourage mucking with them (consider them constants). instead, if you'd like to change colors you should change the _aliases_ like `$pt-text-color`.